ranger/core/loader.py

```diff
@@ -60,10 +61,10 @@ def __init__(self, copy_buffer, do_cut=False, overwrite=False, dest=None,
         self.original_path = dest if dest is not None else self.fm.thistab.path
         self.overwrite = overwrite
         self.make_safe_path = make_safe_path
-        self.percent = 0
+        self.progress = ProgressTracker(self._calculate_size(1))
```

Comment:
The build is failing because the `size` variable is unused. This lead me to this which could use the `size` variable. Or maybe remove the `size` variable and use its definition here.

While looking at the `size` variable definition, I realized I could use the trick with the `max(1, x)` to avoid possible division by zero.

Reply:
The `size` variable was removed. Using the `max(1, x)` trick in ProgressTracker didn't seem to provide any benefits over the `if total_items = 0` condition.